Assets/Scripts/Core/ShipModel/ShipPhysics.cs

```diff
@@ -575,8 +575,8 @@ public class ShipPhysics : MonoBehaviour {
                 if (delta > 0) {
                     var latInputTotal = Mathf.Abs(latH) + Mathf.Abs(latV);
                     if (latInputTotal > 0) {
-                        var latHComponent = Mathf.Abs(latH) / latInputTotal;
-                        var latVComponent = Mathf.Abs(latV) / latInputTotal;
+                        var latHComponent = Mathf.Abs(latH) / latInputTotal * FlightParameters.boostDivertEfficiency;
```

Comment:
so here we're doing the division first then the multiplication and only if the input is greater than 1 so there's a couple things I don't quite get here

* if boostDivertEfficiency is greater than 1 we gain extra thrust from nowhere? (previously this logic steals boost from forward thrust by diverting it into lats). Is this value intended to be less than 1 or left open-ended?
* if you _only_ hold, say, right but not up or down then this logic won't apply anyway right?

Reply:
- |if boostDivertEfficiency is greater than 1 we gain extra thrust from nowhere? (previously this logic steals boost from forward thrust by diverting it into lats). Is this value intended to be less than 1

Its intended to be less than one, but if other people want it higher in order to make a crazy bouncy build, I don't see the harm. 

- if you only hold, say, right but not up or down then this logic won't apply anyway right?

If any of the lats are nonzero, the logic will apply (line 576, 577), but only to the degree that that specific lateral utilized. Boosting with only using the right direction will result in the latVCompenent being multiplied with zero, and the latHComponent being multiplied with something positive.  It is just your old logic but with an extra multiplier to control the magnitude of the effect. The throttle part of the logic is more difficult, but I didn't touch that. I think your logic is good, but it is quite a mind bender with the whole throttle remap thing. If it is wrong, I think we're screwed because I assume this is the logic the current flight system and all the times are based on. I am positive that my change only does what is advertised, tho.